governance/src/test/java/org/apache/servicecomb/governance/LoadBalancerTest.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.servicecomb.governance;
+
+import org.apache.servicecomb.governance.handler.LoadBalanceHandler;
+import org.apache.servicecomb.governance.marker.GovernanceRequest;
+import org.apache.servicecomb.loadbanlance.LoadBalance;
+import org.apache.servicecomb.loadbanlance.LoadBalanceDecorators;
+import org.apache.servicecomb.loadbanlance.LoadBalanceDecorators.LoadBalanceDecorateCheckedSupplier;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+
+@SpringBootTest
+@ContextConfiguration(classes = {GovernanceConfiguration.class, MockConfiguration.class})
+public class LoadBalancerTest {
+  private LoadBalanceHandler loadBalanceHandler;
+
+  @Autowired
+  public void setLoadBalanceHandler(LoadBalanceHandler loadBalanceHandler) {
+    this.loadBalanceHandler = loadBalanceHandler;
+  }
+
+  public LoadBalancerTest() {
+
+  }
+  @Test
+  public void test_loadbalance_random() {
+    LoadBalanceDecorateCheckedSupplier<Object> ds = LoadBalanceDecorators.ofCheckedSupplier(()->"test");
+    GovernanceRequest request = new GovernanceRequest();
+    request.setUri("/loadrandom");
+    request.setServiceName("RandomTest");
+    LoadBalance loadBalance = loadBalanceHandler.getActuator(request);
+    Assert.assertEquals("Random",ds.withFaultInjection(loadBalance));
```

Comment:
I think for load balance do not need this complicity. 


for example

```
    GovernanceRequest request = new GovernanceRequest();
    request.setUri("/loadrandom");
    request.setServiceName("RandomTest");
    LoadBalance loadBalance = loadBalanceHandler.getActuator(request);
    Assert.assertEquals("Random",loadBalance.getRule());
```

This is quit enough